File: lesson5/c_c++/homework/homework5_12.cpp
/*
 * lesson5/c_c++/homework/homework5_12.cpp
 */
#include <iostream>

void primeNumber(int& numb_n) {
    int numb_k = 2;
    while (numb_n % numb_k != 0) {
        ++numb_k;
    }
    if (numb_k == numb_n) {
        std::cout << "prime\n";
    } else {
        std::cout << "no prime\n";
    }
}

int main() {
    int numb;
    std::cout << "Введите число -> ";
    std::cin >> numb;
    primeNumber(numb);
    return 0;
}

File: lesson5/python3/homework/homework5_12.py
#!/usr/bin/python3
#: lesson5/python3/homework/homework5_12.py


def prime_number(numb_p):
    numb_k = 2
    while numb_p % numb_k != 0:
        numb_k += 1
    if numb_k == numb_p:
        print("prime")
    else:
        print("no prime")


numb = int(input("Введите число -> "))
prime_number(numb)

